catalog/glue.go

```diff
@@ -150,8 +151,33 @@ func (c *GlueCatalog) UpdateNamespaceProperties(ctx context.Context, namespace t
 	return PropertiesUpdateSummary{}, fmt.Errorf("%w: [Glue Catalog] update namespace properties", iceberg.ErrNotImplemented)
 }
 
+// ListNamespaces returns a list of Iceberg namespaces from the given Glue catalog.
 func (c *GlueCatalog) ListNamespaces(ctx context.Context, parent table.Identifier) ([]table.Identifier, error) {
-	return nil, fmt.Errorf("%w: [Glue Catalog] list namespaces", iceberg.ErrNotImplemented)
+	params := &glue.GetDatabasesInput{}
+
+	if parent != nil {
+		return nil, fmt.Errorf("hierarchical namespace is not supported")
+	}
```

Comment:
Is this a facet of glue? Or are we just not implementing it yet?

Reply:
It does not support hierarchical namespaces as far as I can tell.